src/fusion_power_plant.cc

```diff
@@ -12,45 +12,71 @@ std::string FusionPowerPlant::str() {
 
 // - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
 void FusionPowerPlant::EnterNotify() {
-  //Pseudocode implementation of EnterNotify()
   cyclus::Facility::EnterNotify();
-  fuel_usage_mass = (burn_rate * (fusion_power / MW_to_GW) / seconds_per_year * context()->dt());
-  fuel_usage_atoms = fuel_usage_mass / tritium_atomic_mass;
-  blanket_turnover = blanket_size * blanket_turnover_rate;
 
-  //Create the blanket material for use in the core
-  const cyclus::Composition::Ptr enriched_li = context->GetRecipe(blanket_inrecipe);
-  cyclus::Material::Ptr blanket = cyclus::Composition::CreateFromAtom(0.0, enriched_li);
+  //fuel_usage_mass = (burn_rate * (fusion_power / MW_to_GW) / 
+  //  seconds_per_year * context()->dt());
+  //fuel_usage_atoms = fuel_usage_mass / tritium_atomic_mass;
+  //blanket_turnover = blanket_size * blanket_turnover_rate;
 
-  //Truncated for readability. It goes on Tritium Storage and gets started, though.
-  fuel_startup_policy.Init(&tritium_storage).Set(fuel_incommod).Start();
-  
-  //Truncated for readability
-  blanket_fill_policy.Init(&blanket).Set(blanket_incommod).Start();
+  //Create the blanket material for use in the core, no idea if this works...
+  const Composition::Ptr enriched_li = context->GetRecipe(blanket_inrecipe);
+  Material::Ptr blanket = Composition::CreateFromAtom(0.0, enriched_li);
```

Comment:
This may work, but this variable will be in local scope and expire when the function ends.  I think `blanket` needs to be a member variable of the class.

Reply:
I think you're right. There was a bit of confusion between the comment that said to move these to here and this (especially since none of this builds anyway, so it's hard to tell what's going to be an issue). I'll add blanket as a member variable, then just set its value here in EnterNotify, similar to how fuel_usage_mass and those other commented out variables work.